dokka-runners/runner-gradle-plugin-classic/src/main/kotlin/org/jetbrains/dokka/gradle/kotlin/KotlinNativeDistributionAccessor.kt

```diff
@@ -27,12 +27,23 @@ internal class KotlinNativeDistributionAccessor(
   project: Project
 ) {
   private val konanDistribution = KonanDistribution(
-    @Suppress("INVISIBLE_MEMBER")
-    project.konanHome
+      // see this comment for the explanation of what's happening:
```

Comment:
minor: may be it will be better to create a top level function like `private fun konanHome(project: Project): String` with both reflection and `alternativeKonanHome` logic (possibly divided there in 2 local functions)?
To keep it the logic of konan home computation in a more clear way?

Reply:
The idea was to minimize changes for review and git blame later on: before, `project.konanHome` was called, and now the same property is being called in the same place, but through reflection; `alternativeKonanHome` didn't exist, so it's added as an additional thing. If this wasn't a bugfix and needed to be maintained in the future - I'd certainly extract the logic, yeah

If it's an eyesore - let me know, I don't mind refactoring it a little like you proposed

Reply:
Thanks for the reasoning, it's fine for me like it's now, no need to change